wgengine/magicsock/endpoint.go

```diff
@@ -591,15 +607,10 @@ func (de *endpoint) startDiscoPingLocked(ep netip.AddrPort, now mono.Time, purpo
 			return
```

Comment:
this exit path means there's a stranded entry in the sentPing map that was never sent and will never be cleaned up